solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -1096,38 +1096,89 @@ public Builder withTheseParamNamesInTheUrl(Set<String> urlParamNames) {
 
     /**
      * Set maxConnectionsPerHost for http1 connections, maximum number http2 connections is limited
-     * by 4
+     * to 4
+     *
+     * @deprecated Please use {@link #withMaxConnectionsPerHost(int)}
      */
+    @Deprecated(since = "9.2")
     public Builder maxConnectionsPerHost(int max) {
+      withMaxConnectionsPerHost(max);
+      return this;
+    }
+    /**
+     * Set maxConnectionsPerHost for http1 connections, maximum number http2 connections is limited
+     * to 4
+     */
+    public Builder withMaxConnectionsPerHost(int max) {
       this.maxConnectionsPerHost = max;
       return this;
     }
 
+    /**
+     * @deprecated Please use {@link #withIdleTimeout(int)}
+     */
+    @Deprecated(since = "9.2")
     public Builder idleTimeout(int idleConnectionTimeout) {
+      withIdleTimeout(idleConnectionTimeout);
+      return this;
+    }
+
+    public Builder withIdleTimeout(int idleConnectionTimeout) {
       this.idleTimeout = idleConnectionTimeout;
       return this;
     }
 
+    /**
+     * @deprecated Please use {@link #setUseHttp1_1(boolean)}
+     */
+    @Deprecated(since = "9.2")
     public Builder useHttp1_1(boolean useHttp1_1) {
+      setUseHttp1_1(useHttp1_1);
+      return this;
+    }
+
+    public Builder setUseHttp1_1(boolean useHttp1_1) {
```

Comment:
How about `withHttp1_1`? not sure `setUseHttp1_1` is an improvement over `useHttp1_1`

Reply:
Or maybe `setHttp1_1`?    or `enableHttp_1_1()` and another `enableHttp_2()` that would flip taht flag?    This one felt a bit odd ot me too!

Reply:
okay, went with `withHttp1_1`...    Looking forward to getting the merged so I can move on to sorting out the time parameters ;-)